Userland/Utilities/less.cpp

```diff
@@ -20,6 +20,8 @@ static struct termios g_save;
 // Flag set by a SIGWINCH signal handler to notify the main loop that the window has been resized.
 static Atomic<bool> g_resized { false };
 
+static Atomic<bool> g_should_switch_buffer { false };
```

Comment:
can you call this something like `g_restore_buffer_on_close`. switch_buffer doesn't really convey when this variable will switch the buffer.